File: Factorial.java
//Anuj Pillai
//201952205
//2-D

import java.util.*;
import java.io.*;

public class Factorial {
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter a number :");
        int n = sc.nextInt();
        int ans = FactorialCalc(n);
        System.out.println("It's factorial will be : "+ans);
    }
    public static int FactorialCalc(int no){
       int fact=1;
       for(int i=1;i<=no;i++){
           fact=fact*i;
       }
       return fact;
    }
}

File: factorial.c
#include <stdio.h>

int fact(int x)
{
    if(x>1)
    x=x*fact(x-1);
    return x;
}

int main()
{
    int num=0;
    printf("Enter the Number for factorial :");
    scanf("%d",&num);
    printf("The value of factorial of %d is : %d",num,fact(num));
    return 0;
}
